content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -1,131 +1,172 @@
-   account and modify the date under :menuselection:`Orders Follow-up --> Last Order Sync`. Pick a
-   date anterior to the last status change of the order that you wish to synchronize and save.
+   When an order is canceled in Amazon, and was already synchronized in Odoo, the corresponding
+   sales order is automatically canceled in Odoo.
+
+Force synchronization
+=====================
+
+In order to force the synchronization of an order, whose status has **not** changed since the
+previous synchronization, start by activating the :ref:`developer mode <developer-mode>`.
+
+Then, navigate to the Amazon account in Odoo (:menuselection:`Sales app --> Configuration -->
+Settings --> Connectors --> Amazon Sync --> Amazon Accounts`), and modify the date under
+:menuselection:`Orders Follow-up --> Last Order Sync`.
+
+Be sure to pick a date that occurs prior to the last status change of the desired order to
+synchronize and save.
 
 .. tip::
-   To synchronize immediately the orders of your Amazon account switch to :ref:`developer mode
-   <developer-mode>`, head to your Amazon account and click on **SYNC ORDERS**. The same can be done
-   with pickings by clicking on **SYNC PICKINGS**.
+   To immediately synchronize the orders of an Amazon account, switch to :ref:`developer mode
+   <developer-mode>`, head to the Amazon account in Odoo, and click :guilabel:`Sync Orders`. The
+   same can be done with pickings by clicking :guilabel:`Sync Pickings`.
 
 Manage deliveries in FBM
 ========================
 
-When a **FBM** (Fulfilled by Merchant) order is synchronized in Odoo, a picking is created along
-with the sales order and the customer. You can either ship all the ordered products to your customer
-at once or ship products partially by using backorders.
+Whenever an FBM (Fulfilled by Merchant) order is synchronized in Odoo, a picking is instantly
+created in the *Inventory* app, along with a sales order and customer. Then, decide to either ship
+all the ordered products to the customer at once, or ship products partially using backorders.
 
-When a picking related to the order is confirmed, a notification is sent to Amazon who will, in
-turn, notify the customer that the order (or a part of it) is on its way.
+When a picking related to the order is confirmed, a notification is then sent to Amazon, who, in
+turn, notifies the customer that the order (or a part of it) is on its way.
 
 .. important::
-   Amazon requires to provide a tracking reference with each delivery. You'll need to assign a
-   carrier. If the carrier doesn't automatically provide a tracking reference, you'll need to set
-   one manually. This concerns all marketplaces.
+   Amazon requires users to provide a tracking reference with each delivery. This is needed to
+   assign a carrier.
+
+   If the carrier doesn't automatically provide a tracking reference, one must be set manually. This
+   rule applies to all marketplaces.
```

Comment:
```suggestion
   rule applies to all marketplaces in Odoo.
```

Reply:
This refers to Amazon marketplaces not Odoo

Reply:
Added in clarification.